bob/cmd.py

```diff
@@ -0,0 +1,127 @@
+#-------------------------------------------------------------------------------
+# bob: cmd.py
+#
+# Command-line functions
+#
+# Dustin J. Mitchell (dustin@v.igoro.us)
+# This code is in the public domain
+#-------------------------------------------------------------------------------
+
+from __future__ import print_function
+
+import os, sys
+import argparse
+
+from bob.compiler import compile_code
```

Comment:
Could you collect the imports from the same module into a single `from module import x,y,z` line?